lib/queue_publisher.rb

```diff
@@ -62,6 +64,29 @@ def publish_message(routing_key, message_data, options = {})
     raise
   end
 
+  def validate_message(message_data)
+    schema = JSON.load(File.read("govuk-content-schemas/message_queue.json"))
```

Comment:
I did originally try using that, but ran in to difficulties as it tried to validate using a sub schema, which didn't work in this case.

I ended up just implementing the necessary behaviour outside of it, but it would be good to improve the SchemaValidator class to make it usable in this case by improving the validation output (which would make using the sub schema unnecessary).